scripts/restore_node.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+protocol=${1:-validator}
+echo ${protocol}
+node_name=$([ $protocol == boot ] && echo "boot_node" || [ $protocol == validator ] && echo "add_validation_node_custom")
```

Comment:
Does it make sense to merge `backup_node` and this script to be able to reuse common parts?